components/mpas-ocean/src/analysis_members/mpas_ocn_moc_streamfunction.F

```diff
@@ -602,14 +604,18 @@ subroutine ocn_compute_moc_streamfunction(domain, timeLevel, err)!{{{
                     sumTransport(k,iTransect) = sumTransport(k,iTransect) + &
                        transectEdgeMaskSigns(currentTransect,iEdge) &
                        * transectEdgeMasks(currentTransect, iEdge) &
-                       * normalVelocity(k,iEdge)*dvEdge(iEdge) &
+                       * (normalMLEvelocity(k,iEdge) + normalVelocity(k,iEdge)) &
```

Comment:
My question from the registry applies here.  Is it worth having a 3D field here that might be all zeros is certain configurations? Or are we sure enough that this field will be non-zero in most of our runs so it's worth carrying around?